tap_frontapp/http.py

```diff
@@ -56,14 +54,20 @@ def request(self, method, path, **kwargs):
                 #so here we just run the request again
                 time.sleep(2)
                 response = requests.request(method, self.url(path), **kwargs)
-
+        elif path == '/conversations' or '/contacts/' in path:
+            #conversations and contacts don't need to produce a report, sleep for shorter
+            response = requests.request(method, self.url(path), **kwargs)
+            time.sleep(.65)
         else:
             response = requests.request(method, self.url(path), **kwargs)
             time.sleep(2)
             response = requests.request(method, self.url(path), **kwargs)
 
-        #print('final3 url=',response.url)
-        self.calls_remaining = int(response.headers['X-Ratelimit-Remaining'])
+        # print('final3 url=',response.url,flush=True)
+        try:
+            self.calls_remaining = int(response.headers['X-Ratelimit-Remaining'])
+        except:
+            time.sleep(2)
```

Comment:
What is the reasoning for this? It could use a comment.
- Does the API sometimes return non-int values for this header? (Resulting in a conversion error to `int`)
    - If so, they should be handled explicitly or documented in a comment.
- Does the API sometimes not return this header? (Resulting in a KeyError on `[]`)
    - In that case, it's less clear what action to take. A sleep may be appropriate, but it all depends on the situation.